odl/discr/grid.py

```diff
@@ -256,6 +256,16 @@ def max_pt(self):
         """
         return np.array([vec[-1] for vec in self.coord_vectors])
 
+    @property
+    def ideg(self):
+        """Indices of the degenerate axes of this grid."""
+        return self.__ideg
+
+    @property
+    def inondeg(self):
+        """Indices of the non-degenerate axes of this grid."""
+        return self.__inondeg
+
```

Comment:
Now that these have found a public use [here](https://github.com/odlgroup/odl/blob/8c82c0b47db1c2b46048982e403e917c5ce10d27/odl/discr/partition.py#L536) I'll keep these public.